js/about/extensions.js

```diff
@@ -32,21 +70,28 @@ class ExtensionItem extends ImmutableComponent {
       this.props.extension.getIn(['manifest', 'icons', '16']) ||
       null
   }
+
   render () {
+    const className = css(
+      commonStyles.listItem,
+      styles.listItem,
+      !this.props.extension.get('enabled') && styles.isDisabled
+    )
     const icon = this.icon
     const permissions = this.props.extension.getIn(['manifest', 'permissions'])
+
     return <div role='listitem'
       disabled={!this.props.extension.get('enabled')}
-      className='listItem'
+      className={className}
       data-extension-id={this.props.extension.get('id')}
       onContextMenu={this.onContextMenu}
       data-context-menu-disable>
-      <div className='extensionImage'>
-        <img src={`${this.props.extension.get('base_path')}/${icon}`} />
+      <div className={css(styles.extensionImage)}>
+        <img className={css(styles.img)} src={`${this.props.extension.get('base_path')}/${icon}`} />
       </div>
       <div className='extensionDetails'>
```

Comment:
can we remove that class or just replace it to `data-id-test` just for future-friendliness?